tests/uvbeam/test_uvbeam.py

```diff
@@ -1990,6 +2002,9 @@ def test_select_frequencies(
         frequencies=freqs_to_keep, freq_chans=chans_to_keep, inplace=False
     )
 
+    print(len(all_chans_to_keep))
+    print(beam2.Nfreqs)
+
```

Comment:
cruft?

Reply:
Yup, removed in bee13af03969accbf712e9db80f799bd0d142a9b